initial "mock" invocation fails because RPMS/x86_64 is not yet a repo

When following the README.md instructions, everything is ok until the first "mock" invocation:

```
[MOCK] RPMS/x86_64/ocaml-oclock-0.3-2.x86_64.rpm
ERROR: Exception(SRPMS/ocaml-oclock-0.3-2.src.rpm) Config(epel-6-x86_64) 0 minutes 1 seconds
ERROR: Command failed: 
 # ['/usr/bin/yum', '--installroot', '/var/lib/mock/epel-6-x86_64/root/', 'groupinstall', 'buildsys-build', '--s]
file:///home/djs/rpmbuild/RPMS/repodata/repomd.xml: [Errno 14] Could not open/read file:///home/djs/rpmbuild/RPMl
Trying other mirror.
Error: Cannot retrieve repository metadata (repomd.xml) for repository: mock. Please verify its path and try agan

make: *** [RPMS/x86_64/ocaml-oclock-0.3-2.x86_64.rpm] Error 30
```

After a manual init:

```
[djs@st30 rpmbuild]$ createrepo --quiet --update RPMS
Could not find valid repo at: /home/djs/rpmbuild/RPMS
```

It works as expected: (so far!)

```
[djs@st30 rpmbuild]$ time make
[MOCK] RPMS/x86_64/ocaml-oclock-0.3-2.x86_64.rpm
[CREATEREPO] RPMS/x86_64/ocaml-oclock-0.3-2.x86_64.rpm
[CURL] SOURCES/squeezed-0.9.0.tar.gz
[RPMBUILD] SRPMS/squeezed-0.9.0-1.src.rpm
[CURL] SOURCES/obuild-0.0.2.tar.gz
[RPMBUILD] SRPMS/ocaml-obuild-0.0.2-0.src.rpm
...
```

--- a/configure.sh
+++ b/configure.sh
@@ -1,7 +1,14 @@
 #!/bin/sh
 
+echo -n "Writing mock configuration..."
 mkdir -p mock
 sed "s|@PWD@|$PWD|" xenserver.cfg.in > mock/xenserver.cfg
 ln -fs /etc/mock/default.cfg mock/
 ln -fs /etc/mock/site-defaults.cfg mock/
 ln -fs /etc/mock/logging.ini mock/
+echo " done"
+
+echo -n "Initializing repository..."
+createrepo --quiet RPMS
+echo " done"
+

